Gemfile

```diff
@@ -3,5 +3,6 @@ source 'https://rubygems.org'
 gemspec
 
 group :release do
-  gem 'github_changelog_generator', '~> 1.16', require: false
+  gem 'faraday-retry', '~> 2.1', require: false
+  gem 'github_changelog_generator', '~> 1.16.4', require: false
```

Comment:
This will not catch 1.17.x, is it intended?

Reply:
yeah. they had some strange problems in the past with minor releases. we pin to that on other gems as well.